pkg/services/ngalert/store/image_test.go

```diff
@@ -90,8 +90,11 @@ func TestIntegrationGetImages(t *testing.T) {
 	assert.Len(t, mismatched, 0)
 	assert.Equal(t, []models.Image{image1}, images)
 
-	// create an image with a URL
-	image2 := models.Image{Path: "https://example.com/example.png"}
```

Comment:
This test was not creating an image with a URL as the comment said, so changed it to use an URL and made it a long one.